### PR TITLE
doc: repo configs

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #1031 

Added general description for repository configs. Each repository will be described letter

### CODE REVIEW

**Review by dgarus — CHANGES_REQUESTED** [review]
@olenagerasimova thanks!
some minor comments.

File: .wiki/Configuration-Repository.md [hunk 1]
```diff
@@ -32,16 +78,29 @@ repo:
...
-    \*:
+    "*":
       - download
 ```
 
+In order to upload a binary file to the storage, send a `PUT` HTTP request with file contents:
+
+```bash
+echo "hello world" > text.txt
+curl -X PUT --data-binary "@text.txt" http://localhost:8080/repo/text.txt
```
[the remark this answers is not in the document]
Comment (olenagerasimova) [reply .wiki/Configuration-Repository.md:89]: I do not think we should mention key here, as this term may be not obvious for thee reader and we can use the name of the just created file here, other parts I've fixed.

**Comment by olenagerasimova** [comment]
@dgarus thanks, corrected

**Review by dgarus — APPROVED** [review]
@olenagerasimova thanks!

**Pull request merged**